client/post-editor/editor-drawer/taxonomies.jsx

```diff
@@ -32,7 +33,11 @@ function isSkippedTaxonomy( postType, taxonomy ) {
 	return false;
 }
 
-function EditorDrawerTaxonomies( { translate, siteId, postType, taxonomies, terms } ) {
+function EditorDrawerTaxonomies( { translate, siteId, postType, isSupported, taxonomies, terms } ) {
+	if ( ! isSupported ) {
+		return null;
```

Comment:
Instead of returning null, perhaps we should show a message about the version of JP being unsupported, and provide a link to upgrade.